compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/node/TreeSet.kt

```diff
@@ -16,7 +16,7 @@
 
 package androidx.compose.ui.node
 
-internal expect class SortedSet<E>(comparator: Comparator<in E>) {
```

Comment:
It's intentional renaming to reduce expectations. Please revert

Reply:
Reverted. Will be upstreamed